prime-dotnet-webapi/Services/interfaces/IEmailService.cs

```diff
@@ -15,12 +15,11 @@ public interface IEmailService
         Task SendSiteApprovedPharmaNetAdministratorAsync(CommunitySite site);
         Task SendSiteApprovedSigningAuthorityAsync(CommunitySite site);
         Task SendSiteRegistrationSubmissionAsync(int siteId, int businessLicenceId, CareSettingType careSettingCode);
+        Task SendHealthAuthoritySiteRegistrationSubmissionDetailsToHIBCAsync(int siteId);
```

Comment:
Can change it, I thought being more specific could help in the future in case we need to send to something other than HIBC it would be good to know where each method is sending. If we rename it then if we send to other places,we can pass the recipient as an argument.

Renamed to `SendHealthAuthoritySiteRegistrationSubmissionAsync`

Reply:
I would argue it's the opposite; specifying who is receiving the email in the method name means you have to change the name if we ever update the recipients. The method name is also incorrect since the email is also being sent to the Prime Support Email. We should avoid putting implementation in our names because then we have to change the names when the implementation changes.